hail/src/main/resources/include/hail/PackDecoder.h

```diff
@@ -0,0 +1,406 @@
+#ifndef HAIL_PACKDECODER_H
+#define HAIL_PACKDECODER_H 1
+
+#include "hail/NativeObj.h"
+#include "hail/ObjectArray.h"
+#include "hail/Region.h"
+#include <unordered_map>
+#include <map>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+
+//#define MYDEBUG 1
+
+//#define BIG_METHOD_INLINE 1
```

Comment:
Should these stay in?

Reply:
My inclination is yes, since they document how to change those settings (while making
it clear that they're disabled by default).